sdk/boms/spring-cloud-azure-dependencies/pom.xml

```diff
@@ -67,6 +67,16 @@
       </dependency>
 
       <!-- Spring Cloud Azure -->
+      <dependency>
+        <groupId>com.azure.spring</groupId>
+        <artifactId>azure-spring-data-cosmos</artifactId>
+        <version>3.33.0-beta.1</version>
+      </dependency>
+      <dependency>
+        <groupId>com.azure.spring</groupId>
+        <artifactId>azure-spring-data-cosmos-test</artifactId>
```

Comment:
This artifact is not necessary if it's only used for test, not used for release.